spec/models/topic_spec.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id    :integer          not null, primary key
+#  label :string           not null
+#
+
+require 'rails_helper'
+describe Topic, type: :model do
+  describe 'topic validations' do
```

Comment:
`describe 'validations' do` is enough since the line above tells that you are describing the `Topic` model